tests/unit/style/stylesheets.rs

```diff
@@ -347,4 +354,45 @@ fn test_report_error_stylesheet() {
     assert_eq!("Unsupported property declaration: 'display: invalid;'", error.message);
     assert_eq!(4, error.line);
     assert_eq!(9, error.column);
+
+    // testing for the url
+    assert_eq!(url, error.url);
+    // assert_eq!(wrong_url, error.url);
+}
+
+
+#[test]
+fn test_report_error_passing_correct_url() {
```

Comment:
I think that the added `assert_eq!`s in the test above may be enough for this actually. so you could probably just remove this test.

@jdm would you still prefer an added test module `error_reporting.rs` that has a custom error reporter?